src/main/scala/beam/agentsim/agents/household/HouseholdFleetManager.scala

```diff
@@ -157,13 +158,15 @@ class HouseholdFleetManager(
         }
       }.getOrElse {
         availableVehicles = availableVehicles match {
-          case firstVehicle :: rest =>
+          //in case of replanning because of TRANSIT failure WALK_TRANSIT is used
+          //but we may want to introduce maxWalkingDistance and check that the agent is close enough to the vehicle
+          case firstVehicle :: rest if atHome(originActivity) =>
```

Comment:
Providing a private vehicle only when the person is at home might be a wrong idea. It may lead to increased number of "Long waling trips".